README.md

```diff
@@ -35,6 +35,12 @@ the review process quicker and more streamlined.
 
 ## Install
 
+To use The Lounge you must have [Node.js](https://nodejs.org/en/download/) installed.
+The oldest Node.js version we support is 4.2.0.
```

Comment:
Where did you get this from? 4.x is the LTS branch, and the newest version of it is the most up to date version of it. Being LTS means that they will provide security updates to it, and bug fixes. That means that each indibidual point release is not "LTS", but the branch itself is LTS.

We aren't going to update this every time there's a new patch to 4.x, so just stick it to 4.0